vars/tmsUpload.groovy

```diff
@@ -94,7 +94,8 @@ void call(Map parameters = [:]) {
 
         def namedUser = jenkinsUtils.getJobStartedByUserId()
 
-        if (config.useGoStep == true) {
+        def useGoStep = config.useGoStep ?: true
+        if (useGoStep != false) {
```

Comment:
Now I see as well that previously it was `config.useGoStep == true`. Let's keep that logic. (Sorry for the back-and-forth)